style-guide/assets/wds-icons-tag-small.svg

```diff
@@ -1,4 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 18 18">
-  <path d="M8.632 14.936l5.304-5.304C14.81 8.758 15.5 7.096 15.5 5.86V4c0-1.384-1.118-2.5-2.5-2.5h-1.86c-1.241 0-2.894.686-3.772 1.564L2.064 8.368a2.51 2.51 0 0 1 .001 3.544l3.023 3.023a2.498 2.498 0 0 1 3.544 0zm-.707-.707a1.5 1.5 0 0 1-2.13-.001l-3.023-3.023a1.51 1.51 0 0 0 0-2.13l5.302-5.304c.692-.69 2.091-1.271 3.066-1.271H13c.83 0 1.5.668 1.5 1.5v1.86c0 .97-.585 2.378-1.271 3.065l-5.304 5.304z"/>
-  <path d="M11.093 7.71a1.802 1.802 0 1 1 0-3.605 1.802 1.802 0 0 1 0 3.604zm0-1a.802.802 0 1 0 0-1.605.802.802 0 0 0 0 1.604z"/>
+<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
All `-small` icons have `<svg width="18" height="18" viewBox="0 0 18 18"`. Please keep the convention.